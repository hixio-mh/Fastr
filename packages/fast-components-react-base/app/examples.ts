import ButtonExamples from "../src/button/examples.data";
export { ButtonExamples };

import CheckboxExamples from "../src/checkbox/examples.data";
export { CheckboxExamples };

import HypertextExamples from "../src/hypertext/examples.data";
export { HypertextExamples };

import DividerExamples from "../src/divider/examples.data";
export { DividerExamples };

import ImageExamples from "../src/image/examples.data";
export { ImageExamples };

import LabelExamples from "../src/label/examples.data";
export {LabelExamples};

import TextField from "../src/text-field/examples.data";
export { TextField };

import ToggleExamples from "../src/toggle/examples.data";
export { ToggleExamples };

import TypographyExamples from "../src/typography/examples.data";
export { TypographyExamples };
