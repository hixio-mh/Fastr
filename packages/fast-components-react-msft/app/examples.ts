import ButtonExamples from "../src/button/examples.data";
export { ButtonExamples };

import CheckboxExamples from "../src/checkbox/examples.data";
export { CheckboxExamples };

import DividerExample from "../src/divider/examples.data";
export { DividerExample };

import FlipperExample from "../src/flipper/examples.data";
export { FlipperExample };

import HeadingExample from "../src/heading/examples.data";
export { HeadingExample };

import ImageExamples from "../src/image/examples.data";
export { ImageExamples };

import LabelExamples from "../src/label/examples.data";
export { LabelExamples };

import ToggleExamples from "../src/toggle/examples.data";
export { ToggleExamples };

import TextFieldExamples from "../src/text-field/examples.data";
export { TextFieldExamples };

import TypographyExamples from "../src/typography/examples.data";
export { TypographyExamples };

import HypertextExamples from "../src/hypertext/examples.data";
export { HypertextExamples };
