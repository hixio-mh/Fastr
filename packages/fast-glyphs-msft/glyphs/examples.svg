<svg width="16" height="16" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="Canvas">
<g id="Group">
<g id="Vector">
<path d="M 4 2.9L 5.6 3.8L 5 4.8L 3.4 3.9L 3.4 5.8L 2.3 5.8L 2.3 3.9L 0.6 4.8L -3.8147e-07 3.8L 1.7 2.9L 0.1 2L 0.7 1L 2.2 1.9L 2.2 2.38419e-08L 3.3 2.38419e-08L 3.3 1.9L 4.9 1L 5.5 2L 4 2.9Z" transform="translate(10.3999 1.09998)"/>
</g>
<g id="Vector_2">
<path d="M 4 2.9L 5.6 3.8L 5 4.8L 3.4 3.9L 3.4 5.8L 2.2 5.8L 2.2 3.9L 0.6 4.8L 4.76837e-08 3.8L 1.6 2.9L 4.76837e-08 2L 0.6 1L 2.2 1.9L 2.2 -1.90735e-07L 3.3 -1.90735e-07L 3.3 1.9L 4.9 1L 5.5 2L 4 2.9Z" transform="translate(1.19995 10.2)"/>
</g>
<g id="Vector_3">
<path d="M 0 8L 0 0L 8 0L 8 8L 0 8ZM 1 1L 1 7L 7 7L 7 1L 1 1ZM 14.4 13.1L 16 14L 15.4 15L 13.8 14.1L 13.8 16L 12.7 16L 12.7 14.1L 11 15.1L 10.4 14.1L 12 13.2L 10.4 12.3L 11 11.3L 12.6 12.2L 12.6 10.3L 13.7 10.3L 13.7 12.2L 15.3 11.3L 15.9 12.3L 14.4 13.1Z"/>
</g>
</g>
</g>
</svg>