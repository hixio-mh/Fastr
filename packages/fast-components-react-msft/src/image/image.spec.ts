import examples from "./examples.data";
import {
    generateSnapshots,
    SnapshotTestSuite,
} from "@microsoft/fast-jest-snapshots-react";
import { ImageProps } from "./index";

describe("image snapshots", (): void => {
    generateSnapshots(examples as SnapshotTestSuite<ImageProps>);
});
