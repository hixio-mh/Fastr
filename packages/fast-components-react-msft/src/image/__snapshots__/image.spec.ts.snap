// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image Image: 0 1`] = `
<picture
  className="picture-0-1-3"
>
  <source
    media="(min-width: 1400px)"
    srcSet="https://placehold.it/600x600/2F2F2F/171717"
  />
  <source
    media="(min-width: 1084px)"
    srcSet="https://placehold.it/600x600/2F2F2F/171717"
  />
  <source
    media="(min-width: 768px)"
    srcSet="https://placehold.it/500x500/2F2F2F/171717"
  />
  <source
    media="(min-width: 540px)"
    srcSet="https://placehold.it/400x400/2F2F2F/171717"
  />
  <source
    media="(min-width: 0)"
    srcSet="https://placehold.it/300x300/2F2F2F/171717"
  />
  <img
    alt="Placeholder with grey background and dimension watermark without any imagery"
    className="image-0-1-1 image_round-0-1-2"
    src="https://placehold.it/300x300/2F2F2F/171717"
  />
</picture>
`;

exports[`image Image: 1 1`] = `
<img
  alt="Placeholder with grey background and dimension watermark without any imagery"
  className="image-0-1-4 image_round-0-1-5"
  sizes="100vw"
  src="https://placehold.it/300x300/2F2F2F/171717"
  srcSet="https://placehold.it/600x600/2F2F2F/171717 2048w, https://placehold.it/600x600/2F2F2F/171717 1778w, https://placehold.it/1600x600/2F2F2F/171717 1399w, https://placehold.it/500x500/2F2F2F/171717 1083w, https://placehold.it/400x400/2F2F2F/171717 767w, https://placehold.it/300x300/2F2F2F/171717 1w"
/>
`;

exports[`image Image: 2 1`] = `
<img
  alt="Placeholder with grey background and dimension watermark without any imagery"
  className="image-0-1-7 image_round-0-1-8"
  sizes={null}
  src="https://placehold.it/300x300/2F2F2F/171717"
  srcSet={null}
/>
`;

exports[`image Image: 3 1`] = `
<picture
  className="picture-0-1-12"
>
  <source
    media="(min-width: 1400px)"
    srcSet="https://placehold.it/600x600/2F2F2F/171717"
  />
  <source
    media="(min-width: 1084px)"
    srcSet="https://placehold.it/600x600/2F2F2F/171717"
  />
  <source
    media="(min-width: 768px)"
    srcSet="https://placehold.it/500x500/2F2F2F/171717"
  />
  <source
    media="(min-width: 540px)"
    srcSet="https://placehold.it/400x400/2F2F2F/171717"
  />
  <source
    media="(min-width: 0)"
    srcSet="https://placehold.it/300x300/2F2F2F/171717"
  />
  <img
    alt="Placeholder with grey background and dimension watermark without any imagery"
    className="image-0-1-10"
    src="https://placehold.it/300x300/2F2F2F/171717"
  />
</picture>
`;

exports[`image Image: 4 1`] = `
<img
  alt="Placeholder with grey background and dimension watermark without any imagery"
  className="image-0-1-13"
  sizes="100vw"
  src="https://placehold.it/539x300/2F2F2F/171717"
  srcSet="https://placehold.it/2048x600/2F2F2F/171717 2048w, https://placehold.it/1778x600/2F2F2F/171717 1778w, https://placehold.it/1399x600/2F2F2F/171717 1399w, https://placehold.it/1083x500/2F2F2F/171717 1083w, https://placehold.it/767x400/2F2F2F/171717 767w, https://placehold.it/539x300/2F2F2F/171717 1w"
/>
`;

exports[`image Image: 5 1`] = `
<img
  alt="Placeholder with grey background and dimension watermark without any imagery"
  className="image-0-1-16"
  sizes={null}
  src="https://placehold.it/539x300/2F2F2F/171717"
  srcSet={null}
/>
`;
